### PR TITLE
bugfix: CLDSRV-56 Do not encode tokens again

### PR DESCRIPTION
**Pull request by miniscruff**
There is also the alternative approach of double decoding. We could also `while (token.contains('%')) token=decode(token)` to for sure get rid of extra encoding layers...

### CODE REVIEW

**Comment by jonathan-gramain** [comment]
@miniscruff target should be `development/7.4` I believe

**Comment by naren-rajendran** [comment]
No tests? otherwise LGTM.

**Comment by miniscruff** [comment]
> No tests? otherwise LGTM.

working on it now

**Review by anurag4DSB — APPROVED** [review]

**Comment by miniscruff** [comment]
> No tests? otherwise LGTM.

Ok tests are in, will resolve conflicts in the morning I changed the base branch to 7.4.

File: tests/unit/api/bucketGet.js [hunk 1]
```diff
@@ -108,6 +107,25 @@ const tests = [
...
+            },
+            baseGetRequest
+        ),
+        assertion: result => {
+            assert.strictEqual(result.ListBucketResult.Contents[0].Key[0],
+                objectName3);
+            assert.strictEqual(result.ListBucketResult.Contents[1], undefined);
+            assert.strictEqual(
+                result.ListBucketResult.NextContinuationToken[0],
+                'aW52YWxpZFVSSX5+fmI='
```
Comment (naren-rajendran) [inline tests/unit/api/bucketGet.js:144]: would this be the value of 'next continuation token' every time?

Comment (miniscruff) [reply tests/unit/api/bucketGet.js:144]: the value is the base64 encoding of objectName3

**Review by naren-rajendran — APPROVED** [review]

**Review by nicolas2bert — APPROVED** [review]

**Pull request merged**